scripts/loader.cmd

```diff
@@ -76,9 +86,14 @@ FOR /L %%i IN (1,1,%MAX_RETRIES%) DO (
         ECHO Rebooting host
         SHUTDOWN /R
         EXIT /B 0
+    ) ELSE (
+    rem normal exit code when using ctrl+c
+    IF !KERNEL_EXIT_CODE! EQU 130 (
```

Comment:
Under what scenario does kernel exit 130? We're not handling 130 on linux.

Reply:
As the comment says, this is a normal exit when using ctrl+c to exit. The Windows service uses ctrl+c to shutdown Greengrass and we do not want to retry in that case.